packages/taquito-signer/src/derivation-tools/ed25519.ts

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable @typescript-eslint/no-this-alias */
+import { HMAC } from "@stablelib/hmac";
```

Comment:
we should stick to single quotes, here and a few other files. Running the linter should fix it